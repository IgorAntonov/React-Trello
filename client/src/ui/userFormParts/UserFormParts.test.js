import React from 'react';
import { shallow } from 'enzyme';

import {
  StyledForm,
  Label,
  StyledField,
  Wrapper,
  FormActions,
  Submit,
  Cancel,
  SubmitError
} from './UserFormParts';


test('Userform parts should match snapshots', () => {
  expect(shallow(<StyledForm />)).toMatchSnapshot();
  expect(shallow(<Label />)).toMatchSnapshot();
  expect(shallow(<StyledField />)).toMatchSnapshot();
  expect(shallow(<Wrapper />)).toMatchSnapshot();
  expect(shallow(<FormActions />)).toMatchSnapshot();
  expect(shallow(<Submit />)).toMatchSnapshot();
  expect(shallow(<Cancel />)).toMatchSnapshot();
  expect(shallow(<SubmitError />)).toMatchSnapshot();
});

