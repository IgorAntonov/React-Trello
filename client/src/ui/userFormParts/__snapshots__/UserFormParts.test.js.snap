// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Userform parts should match snapshots 1`] = `
<Form
  className="UserFormParts__StyledForm-hzDVqG hmDNBc"
/>
`;

exports[`Userform parts should match snapshots 2`] = `
<label
  className="UserFormParts__Label-PFUIj cACuHb"
/>
`;

exports[`Userform parts should match snapshots 3`] = `
<Field
  className="UserFormParts__StyledField-gRCJyV kRDSgF"
/>
`;

exports[`Userform parts should match snapshots 4`] = `
<div
  className="UserFormParts__Wrapper-kpSPIp vSvWc"
/>
`;

exports[`Userform parts should match snapshots 5`] = `
<div
  className="UserFormParts__FormActions-jXKkKl dYijO"
/>
`;

exports[`Userform parts should match snapshots 6`] = `
<Button
  className="UserFormParts__Submit-gioWXW danNXL"
/>
`;

exports[`Userform parts should match snapshots 7`] = `
<StyledLink
  className="UserFormParts__Cancel-hJQiUA hnGbDz"
/>
`;

exports[`Userform parts should match snapshots 8`] = `
<P
  className="UserFormParts__SubmitError-bmJoMZ hcxXxq"
/>
`;
